src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32Program.cs

```diff
@@ -976,35 +887,22 @@ public static Win32Program[] All(ProgramPluginSettings settings)
 
             try
             {
-                var programs = new List<Win32Program>().AsParallel();
-
-                var unregistered = UnregisteredPrograms(settings.ProgramSources, settings.ProgramSuffixes);
```

Comment:
Can you elaborate on why `UnregisteredPrograms` is deleted?

Reply:
This is also functionality which is not yet enabled in PT run (add custom indexing paths)